src/twisted/trial/_dist/workertrial.py

```diff
@@ -62,9 +62,29 @@ def main(_fdopen=os.fdopen):
     """
     Main function to be run if __name__ == "__main__".
 
+    This is called in the sub-process spawned by L{DistTrialRunner}.
+
     @param _fdopen: If specified, the function to use in place of C{os.fdopen}.
     @type _fdopen: C{callable}
     """
+    if _fdopen == os.fdopen:
```

Comment:
happy to receive suggestion for how to better write this code.

Maybe instead of `if _fdopen == os.fdopen:` we can have `if _connectStandardPipes:` where `_connectStandardPipes` is an argument to main which is false by default, but is set to true inside `if __name__ == "__main__":`

Like the code from here https://github.com/twisted/twisted/pull/1571/files#diff-a85551b557408ab05e55cdb2af4ab7e930bd039cc1b9c7daf3d91d722ce9f610R97